README.md

```diff
@@ -222,6 +222,28 @@ In order to do that, just call:
 openAds.resetConnectors()
 ```
 
+## Logging
+
+In order to debug what's happening with the Ads Loading, OpenAds uses [LogLevel](https://github.com/pimterry/loglevel) as it's logging framework.
+Also, it's using the [LogLevel Plugin Prefix](https://github.com/kutuluk/loglevel-plugin-prefix) plugin to configure how the log messages will be displayed.
+
+To configure it, you can set this node into the **config** object when initializing OpenAds:
```

Comment:
In order to set the Logger up, add 'LogLevel' node entry into the **config** object when calling OpenAds 'init' method as follows:

Reply:
:+1: